### ISSUE TITLE
Incorrect return type in header files causing Doxygen to fail

### ISSUE THREAD

**Issue by dmtrinh** [mit-dci/opencbdc-tx#52]
### Affected Branch

trunk

### Basic Diagnostics

- [X] I've pulled the latest changes on the affected branch and the issue is still present.

- [ ] The issue is reproducible in docker

### Description

A few header files have extraneous /return commands causing Doxygen to fail.  These appear to be innocent typos due to copy/paste.

In order to reproduce the issue, follow these steps:

1.   Clone repo
```
git clone --recurse-submodules https://github.com/mit-dci/opencbdc-tx
```
2.   Run Doxygen
```
doxygen Doxyfile
```
The errors you'll eventually see are:
```
...
Generating call graph for function cbdc::transaction::wallet::seed
/Users/duc/Dev2/opencbdc-tx_dmt/src/uhs/transaction/wallet.hpp:142: error: documented empty return type of cbdc::transaction::wallet::seed_readonly (warning treated as error, aborting now)
...
Generating call graph for function cbdc::watchtower::error_cache::check_tx_id
/Users/duc/Dev2/opencbdc-tx_dmt/src/uhs/atomizer/watchtower/error_cache.hpp:36: error: documented empty return type of cbdc::watchtower::error_cache::push_errors (warning treated as error, aborting now)
Exiting...


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by HalosGhost** [comment]
Confirmed! Thank you for reporting!

**Comment by HalosGhost** [comment]
closed by #53
